public-ui/src/app/utils/db-config.service.ts

```diff
@@ -15,7 +15,9 @@ export class DbConfigService {
   /* CONSTANTS */
   MALE_GENDER_ID = '8507';
   FEMALE_GENDER_ID = '8532';
-  OTHER_GENDER_ID = '8521';
+  // OTHER_GENDER_ID = '8521';
+  // Current data has 0 for other gender.
```

Comment:
Huh - is this expected? This seems like it could be a data quality issue - is there a Jira ticket with more context?

Reply:
Will open one.